vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ZoteroDefaultMetaData.java

```diff
@@ -0,0 +1,118 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import edu.asu.diging.vspace.core.model.IReference;
+
+public class ZoteroDefaultMetaData implements IReferenceMetadata {
+
+    String biblioMetadataStyle = "defaultZoteroMetadata";
+
+    //For now its default
+    ICitationStyle citationStyle;
+
+    IReference reference;
+    
+    //These all values are fixed for default metadata 
+    String urlVersion = "url_ver=Z39.88-2004";
+    
+    String ctxVersion = "ctx_ver=Z39.88-2004";
+    
+    String rfrId = "info:sid/zotero.org:2";
+    
+    String rftValFmt = "info:ofi/fmt:kev:mtx:dissertation";
```

Comment:
these variables should be private or is there a reason they are not?

Reply:
Done